.travis.yml

```diff
@@ -81,6 +81,8 @@ script:
   - |
     if [[ -v MAVEN_CHECKS ]]; then
       ./mvnw install -DskipTests -B -T C1 -P ci
+      PRESTO_VERSION=`mvn -Dexec.executable='echo' -Dexec.args='${project.version}' --non-recursive exec:exec -q`
+      docker build . --build-arg PRESTO_VERSION=${PRESTO_VERSION} -t presto:${PRESTO_VERSION}
```

Comment:
Please verify that when `./mvnw install -DskipTests -B -T C1 -P ci` fails, the travis build fails too.

i think it won't fail. I'd write this in separate commands:
```
  - |
    if [[ -v MAVEN_CHECKS ]]; then
      ./mvnw install -DskipTests -B -T C1 -P ci
    fi
  - |
    if [[ -v MAVEN_CHECKS ]]; then
      .... docker ....
    fi
```

Reply:
@findepi This links to the larger question above about wether this should be part of the Maven build or should be standalone. Do you have an opinion there?

Reply:
I don't see yet how we will use this Dockerfile. Would be great to publish it automatically to docker hub, for each released Presto version.
Perhaps we can harness Travis to do that. Maybe something like https://github.com/findepi/graalvm-docker/blob/master/.travis.yml#L15-L19 (but for release tags instead of `master`)

From this perspective, I don't think plugging Dockerfile into Maven build is going to be helpful in the long run.

cc @electrum @kokosing

Reply:
The Docker file/container serves three purposes: 
* Provide a Docker container for Presto users to deploy on their own infrastructure. 
* As a starting point for others looking to utilize or modify a Presto image for their own use.
* A base image for the Presto operator which is in progress.

Additionally I agree that we should automatically publish a container for each Presto release and I like the idea of adding it as part of the Travis release process, however I'm at a loss of how to accomplish that at the moment. The presto deployment process is much more complicated than the example provided.

Reply:
@11xor6 right, let's consider automatic publication out of scope for this PR.